### PR TITLE
[feature][connector-v2] add sqlServer CDC

### PR DESCRIPTION
**Pull request by ic4y**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Comment by hailin0** [comment]
Please check code style

File: seatunnel-examples/seatunnel-engine-examples/src/main/java/org/apache/seatunnel/example/engine/SeaTunnelEngineExample.java [hunk 1]
```diff
@@ -32,7 +32,8 @@
 public class SeaTunnelEngineExample {
 
     public static void main(String[] args) throws FileNotFoundException, URISyntaxException, CommandException {
-        String configFile = getTestConfigFile("/examples/fake_to_console.conf");
+        String configFile = getTestConfigFile("/examples/sqlservercdc_to_console.conf");
+//        String configFile = getTestConfigFile("/examples/mysqlcdc_to_console.conf");
         ClientCommandArgs clientCommandArgs = new ClientCommandArgs();
```
Comment (hailin0) [inline seatunnel-examples/seatunnel-engine-examples/src/main/java/org/apache/seatunnel/example/engine/SeaTunnelEngineExample.java]: add `mysqlcdc_to_console.conf` `sqlservercdc_to_console.conf` into `resources/`

File: plugin-mapping.properties [hunk 2]
```diff
@@ -160,6 +160,9 @@ seatunnel.source.Notion = connector-http-notion
 seatunnel.sink.RabbitMQ = connector-rabbitmq
 seatunnel.source.RabbitMQ = connector-rabbitmq
 seatunnel.source.OpenMldb = connector-openmldb
+seatunnel.source.MySQL-CDC = connector-cdc-mysql
+seatunnel.source.SqlServer-CDC = connector-cdc-sqlserver
 seatunnel.source.Maxcompute = connector-maxcompute
 seatunnel.sink.Maxcompute = connector-maxcompute
-seatunnel.source.MySQL-CDC = connector-cdc-mysql
\ No newline at end of file
+seatunnel.source.MySQL-CDC = connector-cdc-mysql
```
Comment (hailin0) [inline plugin-mapping.properties]:
```suggestion
seatunnel.source.MySQL-CDC = connector-cdc-mysql
seatunnel.source.SqlServer-CDC = connector-cdc-sqlserver
```

File: plugin-mapping.properties [hunk 3]
```diff
@@ -160,6 +160,9 @@ seatunnel.source.Notion = connector-http-notion
 seatunnel.sink.RabbitMQ = connector-rabbitmq
 seatunnel.source.RabbitMQ = connector-rabbitmq
 seatunnel.source.OpenMldb = connector-openmldb
+seatunnel.source.MySQL-CDC = connector-cdc-mysql
+seatunnel.source.SqlServer-CDC = connector-cdc-sqlserver
```
Comment (hailin0) [inline plugin-mapping.properties]:
```suggestion
```

**Review by hailin0 — COMMENTED** [review]
@ic4y  Please fix conflict

**Review by hailin0 — APPROVED** [review]

**Review by ashulin — APPROVED** [review]

**Pull request merged**